README.md

```diff
@@ -1 +1,55 @@
-# HTML-Form_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# EMBEDDING IMAGES AND VIDEO
+
+> This is a Microverse project where we have to create a replica of INTUIT signup page.
+
+![screenshot](./images/screenshot.png)
+
+This is INTUIT signup page. In this project, we also setup linters to help fast track errors. The website is not responsive yet as it is only a practice project.
+
+
+## Built With
+
+- Major languages: HTML & CSS
+- Tool: Linter
+- Frameworks: None
+
+## Live Demo
+
+[Live Demo Link](https://kingobaino1.github.io/HTML-Form_Project/.)
+
+
+## Getting Started
+
+**Log on to the Github and clone the repository to your local machine.**
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] To make your readme file more professional, you guys can add your names here